resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -128,13 +142,22 @@ def main(args):
 
             if args.no_control:
                 if header_indices["TUMOR_COL"] == -1:
-                    header_indices["TUMOR_COL"] = 9
+                    if args.skipOrderCheck: 
+                        header_indices["TUMOR_COL"] = 9
```

Comment:
What is the actual name of the column, if no_control == True and "TUMOR_COL" does not exist? Would it not be better to do something like

```python
header_indices["TUMOR_COL"] == header_indices["BLI_BLA_BLUB_COL"]
```
Otherwise you are again just assuming that column 9 is the TUMOR_COL (though named differently). This would be like not using the header indices at all.

Reply:
When `skipOrderCheck` is true, user already verified that the column 10 contains tumor genotype, then this good. And also the `header_indices` dict is only storing the indices here. And I will also add a check for the number of columns in the no_control settings.